### PR TITLE
BD-2163: /purchases/revenue_series and /purchases/quantity_series endpoint docs

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2163**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Comment by lydia-xie** [comment]
@cla-bot check

File: _docs/_api/endpoints/export/purchases/get_number_of_purchases.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Rate limit
+
+{% multi_lang_include rate_limits.md endpoint='purchases product list' %}
+
+## Request parameters
+
+| Parameter | Required | Data Type | Description |
+|---|---|---|---|
+| `ending_at` | Optional | Datetime (ISO-8601 string) | Date on which the data export should end. Defaults to time of the request. |
```
Comment (cjkarl) [inline _docs/_api/endpoints/export/purchases/get_number_of_purchases.md:27]: I think I've seen ISO-8601 be a link similar to the ending_at param here: https://www.braze.com/docs/api/endpoints/export/campaigns/get_campaign_analytics/

Similar comment for the other endpoint.

File: _docs/_api/endpoints/export/purchases/get_number_of_purchases.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| `length` | Required | Integer | Maximum number of days before `ending_at` to include in the returned series. Must be between 1 and 100 (inclusive). |
+| `unit` | Optional | String | Unit of time between data points. Can be day or hour, defaults to day. |
+| `app_id` | Optional | String | App API identifier retrieved from the **API Keys** page. If excluded, results for all apps in a workspace will be returned. |
```
Comment (cjkarl) [inline _docs/_api/endpoints/export/purchases/get_number_of_purchases.md:30]: I intended the API Keys to be a link to a help page for the API keys: I was mainly copying https://www.braze.com/docs/api/endpoints/export/kpi/get_kpi_mau_30_days/#rate-limit

Similar comment for the other endpoint

File: _docs/_api/endpoints/export/purchases/get_number_of_purchases.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  “data”: [{
+        "time": "2023-04-03",
+        "purchase_quantity": 118
+      },
+      {
+        "time": "2023-04-04",
+        "purchase_quantity": 127
+      }],
+  “message”: “success”
+}
```
Comment (cjkarl) [inline _docs/_api/endpoints/export/purchases/get_number_of_purchases.md:56]: My bad, I didn't realize there was an established way to show these example responses, I think this falls in line more with what other pages are doing: 
```
{
    "message": (required, string) the status of the export, returns 'success' when completed without errors,
    "data" : [
        {
            "time" : (string) the date as ISO 8601 date,
            "purchase_quantity" : (int) the number of items purchased in the time period
        },
        ...
    ]
}
```

File: _docs/_api/endpoints/export/purchases/get_revenue_series.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+{
+  “data”: [{
+        "time": "2023-04-03",
+        "revenue": 1186.99
+      },
+      {
+        "time": "2023-04-04",
+        "revenue": 1273.98
+      }],
+  “message”: “success”
```
Comment (cjkarl) [inline _docs/_api/endpoints/export/purchases/get_revenue_series.md:55]: Similar to above: 
```
{
    "message": (required, string) the status of the export, returns 'success' when completed without errors,
    "data" : [
        {
            "time" : (string) the date as ISO 8601 date,
            "revenue" : (int) amount of revenue for the time period
        },
        ...
    ]
}
```

File: _docs/_api/endpoints/export/purchases/get_number_of_purchases.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+{% multi_lang_include rate_limits.md endpoint='purchases product list' %}
+
+## Request parameters
+
+| Parameter | Required | Data Type | Description |
+|---|---|---|---|
+| `ending_at` | Optional | Datetime ([ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) string) | Date on which the data export should end. Defaults to time of the request. |
+| `length` | Required | Integer | Maximum number of days before `ending_at` to include in the returned series. Must be between 1 and 100 (inclusive). |
+| `unit` | Optional | String | Unit of time between data points. Can be day or hour, defaults to day. |
+| `app_id` | Optional | String | App API identifier retrieved from the [**API Keys**]({{site.baseurl}}/user_guide/administrative/app_settings/api_settings_tab/) page. If excluded, results for all apps in a workspace will be returned. |
```
Comment (bre-fitzgerald) [inline _docs/_api/endpoints/export/purchases/get_number_of_purchases.md:30]:
```suggestion
| `app_id` | Optional | String | App API identifier retrieved from the [API Keys]({{site.baseurl}}/user_guide/administrative/app_settings/api_settings_tab/) page. If excluded, results for all apps in a workspace will be returned. |
```

File: _docs/_api/endpoints/export/purchases/get_revenue_series.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+{% multi_lang_include rate_limits.md endpoint='purchases product list' %}
+
+## Request parameters
+
+| Parameter | Required | Data Type | Description |
+|---|---|---|---|
+| `ending_at` | Optional | Datetime ([ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) string) | Date on which the data export should end. Defaults to time of the request. |
+| `length` | Required | Integer | Maximum number of days before `ending_at` to include in the returned series. Must be between 1 and 100 (inclusive). |
+| `unit` | Optional | String | Unit of time between data points. Can be day or hour, defaults to day. |
+| `app_id` | Optional | String | App API identifier retrieved from the [**API Keys**]({{site.baseurl}}/user_guide/administrative/app_settings/api_settings_tab/) page. If excluded, results for all apps in a workspace will be returned. |
```
Comment (bre-fitzgerald) [inline _docs/_api/endpoints/export/purchases/get_revenue_series.md:30]:
```suggestion
| `app_id` | Optional | String | App API identifier retrieved from the [API Keys]({{site.baseurl}}/user_guide/administrative/app_settings/api_settings_tab/) page. If excluded, results for all apps in a workspace will be returned. |
```

File: _docs/_api/endpoints/export/purchases/get_revenue_series.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+      },
+    ...
+  ]
+}
+```
+
+{% endapi %}
+
+{% alert tip %}
+For help with CSV and API exports, visit [Export troubleshooting](https://www.braze.com/docs/user_guide/data_and_analytics/export_braze_data/export_troubleshooting/).
```
Comment (bre-fitzgerald) [inline _docs/_api/endpoints/export/purchases/get_revenue_series.md:61]:
```suggestion
For help with CSV and API exports, visit [Export troubleshooting]({{site.baseurl}}/user_guide/data_and_analytics/export_braze_data/export_troubleshooting/).
```

File: _docs/_api/endpoints/export/purchases/get_number_of_purchases.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+      },
+    ...
+  ]
+}
+```
+
+{% endapi %}
+
+{% alert tip %}
+For help with CSV and API exports, visit [Export troubleshooting](https://www.braze.com/docs/user_guide/data_and_analytics/export_braze_data/export_troubleshooting/).
```
Comment (bre-fitzgerald) [inline _docs/_api/endpoints/export/purchases/get_number_of_purchases.md:61]:
```suggestion
For help with CSV and API exports, visit [Export troubleshooting]({{site.baseurl}}/user_guide/data_and_analytics/export_braze_data/export_troubleshooting/).
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Few minor comments for your review 👍

**Review by bre-fitzgerald — APPROVED** [review]

**Comment by lydia-xie** [comment]
@cla-bot check

**Pull request merged**